.changeset/tidy-bags-dig.md

```
@@ -0,0 +1,17 @@
+---
+'@dexto/webui': patch
+'@dexto/core': patch
+'dexto': patch
+---
+
+Add blob storage system for persistent binary data management:
+  - Implement blob storage backend with local filesystem support
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)